components/aboutMe/aboutMe.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { View, Text, StyleSheet, ScrollView, Image } from 'react-native';
+
+const AboutMe = () => {
+    return (
+        <ScrollView style={styles.ScrollView}>
+            <View style={styles.body}>
+                <Text style={styles.titleText}>About me</Text>
+                <Text style={styles.mainText}>Hello. I'm Max. I'm 25.I was born in Republic of Belarus.
+                    My favorite hobby is learning JS and react-redux, relax with my friends.I have proffesional
+                    driver's license. I graduated Belarussian National Technical University by speciality
+                    "Design and production sports equipments".
+                </Text>
+                <Text style={styles.titleText}>My skills</Text>
+                <View style={styles.skills} >
+                    <View style={styles.blockItem}>
```

Comment:
you have here 4 items. Maybe better will be to create an array of links and to make a map of it? 
`arrayOfLinks.map(link => <View>
<Image source={{ uri: link }} />
</View>)`

you can define such array in AboutMe.constants.js file and import here in render.

Reply:
AboutMe.constants.js

const ARRAY_OF_LINKS = [https://s1.iconbird.com/ico/2013/9/446/w512h5121380376407MetroUIHTML5.png, ..., https://s1.iconbird.com/ico/2013/9/446/w512h5121380376407MetroUIHTML5.png]